composeApp/src/commonMain/kotlin/dev/upvote/presentation/product/ProductScreen.kt

```diff
@@ -27,80 +27,184 @@ import dev.upvote.presentation.error.ShowError
 import dev.upvote.ui.ExpandableSearchView
 
 
+//@Composable
```

Comment:
I would remove these lines rather than comment them out. We have `git` history in case we need it again